driftbase/models/db.py

```diff
@@ -356,8 +389,10 @@ class Match(ModelBase):
     status_date = Column(DateTime, nullable=True)
     unique_key = Column(String(50), nullable=True)
 
-
 class MatchPlayer(ModelBase):
+    """
+    The match that a player is currently in
```

Comment:
This comment is incorrect, one entry is created for every player in every match, but only one with a given match/player combo should exist.